utilities/aap.py

```
@@ -0,0 +1,619 @@
+    when triggering AAP hooks during migration.
+
+    Args:
+        ocp_admin_client: OpenShift admin client.
+        fixture_store: Fixture store for resource tracking.
+        namespace: MTV operator namespace (e.g., openshift-mtv).
+        token: AWX OAuth2 token string.
+
+    Returns:
+        Secret: The created Kubernetes Secret.
+    """
+    LOGGER.info(f"Creating AAP token secret '{AAP_TOKEN_SECRET_NAME}' in namespace '{namespace}'")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)